app/locations/views/locations.njk

```diff
@@ -30,7 +30,7 @@
           {% for location in locations %}
             <li>
                 {% if location.type == "regions" %}
-                  <a href="/locations/{{ location.id }}?isRegion=true">
+                  <a href="/locations/region/prisons/{{ location.id }}">
                     {{ location.key }} {{ location.name }}
```

Comment:
Would it be possible to move the key outside of the link a bit more like the prototype please:

![image](https://user-images.githubusercontent.com/3327997/86108195-c630ed00-baba-11ea-8232-b929486c0da1.png)

I also wonder whether we can group the regions separately so that as we discussed, if a user has both permissions they still have the ability to navigate via a single location.

Maybe two separate blocks like this:

![image](https://user-images.githubusercontent.com/3327997/86108755-79014b00-babb-11ea-8a30-d92732b66d56.png)

Reply:
I didn't notice the difference, my bad. Moving the key outside.

As for the grouping, I asked and the answer is that it won't happen in this story. However, we have a future story designed to address the issue.

Reply:
So the problem with currently only showing one or the other is that if a person how both a PMU role and any other kind of role, they can't use the rest of the service because the regions are only used for allocations. For example, you can't now view outgoing or incoming moves for a single location because it is only possible to select a region.

It also wouldn't be possible to create a new move because those journeys rely on a current location being set.

Would it be much work to change the nesting of the if on this template to just wrap regions and always show locations instead of it being an if/else?

Reply:
Unfortunately, I don't have enough context to make an informed decision (the proposed change seems small, though).
Also, the proposal changes the scope of the story and should be discussed with the rest of the team. I copy-pasted the comment into the Jira ticket so that we can keep the conversation going.

Reply:
Thanks. Have also followed up on the ticket to help with the conversation. I think my concern comes from knowing that there at, at the very least, test accounts (my own included) that contain both PMU and other roles. 

And this current implementation would mean that it would be harder for me to use the other parts of the service like creating a move if I sign in with that account. The change seems small to support both and I think that's a nicer approach moving forward. That permissions should add/remove things in most cases instead of completely changing feature behaviour.

Reply:
I implemented the list with your design and not how the ticket described.
I left a comment in the ticket.